Makefile

```diff
@@ -0,0 +1,39 @@
+APPS = asset-manager content-store govuk-content-schemas multipage-frontend \
+	publishing-api router router-api rummager specialist-frontend \
+	specialist-publisher static travel-advice-publisher
+
+all: test
```

Comment:
do we not want these to be distinct? eg you want to just run the tests and not have it go all the way back to cloning it? I'd have thought only in `all` would you want the dependency to kick in.